diff --git a/interdrv/v2/cvi_vc_drv/Makefile b/interdrv/v2/cvi_vc_drv/Makefile
index 2cb2351..f6c7f2c 100644
--- a/interdrv/v2/cvi_vc_drv/Makefile
+++ b/interdrv/v2/cvi_vc_drv/Makefile
@@ -29,7 +29,7 @@ VCODEC_DRIVER_HEADER := -I$(PWD)/	\
 			-I$(PWD)/$(JPEG_DRIVER_PATH)/cvi_jpg_interface/inc/	\
 			-I$(PWD)/$(VCODEC_MODULE_PATH)/include/
 
-cvi_vc_driver-objs :=	$(COMMON_PATH)/rcKernel/cvi_rc_kernel.o	\
+soph_vc_driver-objs :=	$(COMMON_PATH)/rcKernel/cvi_rc_kernel.o	\
 			$(COMMON_PATH)/rcKernel/cvi_float_point/cvi_float_point.o	\
 			$(COMMON_PATH)/rcKernel/cvi_float_point/cvi_soft_float.o	\
 			$(VCODEC_DRIVER_PATH)/sample/main_enc_cvitest.o	\
@@ -80,7 +80,7 @@ cvi_vc_driver-objs :=	$(COMMON_PATH)/rcKernel/cvi_rc_kernel.o	\
 			cvi_vc_drv.o	\
 			cvi_vc_drv_proc.o
 
-obj-m = cvi_vc_driver.o
+obj-m = soph_vc_driver.o
 
 ccflags-y += -I$(PWD)/../include/common/uapi/ -I$(PWD)/../include/chip/$(CVIARCH_L)/uapi/
 ccflags-y += -I$(PWD)/../include/common/kapi/
